spinalcordtoolbox/gui/widgets.py

```diff
@@ -10,6 +10,7 @@
 from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT as NavigationToolbar
 from matplotlib.figure import Figure
 from matplotlib.widgets import Cursor
+from skimage import exposure
```

Comment:
Importing `skimage.exposure` and using the fully qualified function name would be more explicit.
But that's okay.

Reply:
fixed in d5c4b017289f742ff52562cebdc264727446543b